hack/check-copyright.sh

```diff
@@ -6,7 +6,7 @@ nocopyright=""
 for f in $files
 do
         # For each file, check if it contains "Copyright" in the first three lines.
```

Comment:
nit: should we update the comment ('three' -> 'four')?

Reply:
yep we should, although this change needs to be made in calico not here (since this is just a mirror)